lib/chibi/test.scm

```diff
@@ -23,6 +23,20 @@
             args)
   (newline (current-error-port)))
 
+(define (exception-message exc)
+  (let* ((s (parameterize
+               ((current-output-port (open-output-string)))
+             (print-exception exc (current-output-port))
+             (get-output-string (current-output-port))))
+         (n (- (string-length s) 1)))
+    (let loop ((i 0))
```

Comment:
Please add a comment: "Strip the ERROR: prefix if present."

Reply:
Implemented